packages/code-provider/src/types.ts

```
@@ -92,6 +92,13 @@ export interface CopyFilesInput {
 }
 export interface CopyFileOutput {}
 
+export interface CreateDirectoryInput {
+    args: {
+        path: string;
+    };
+}
+export interface CreateDirectoryOutput {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)